lib/queries/crud.js

```diff
@@ -16,16 +16,26 @@ export default function ({
     specificSorts,
     groupByCols,
     withQuery,
+    permanentFilters = {},
     allowPrimaryKeyUpdate,
 }) {
     const removeOne = removeOneQuery({ table, primaryKey });
```

Comment:
permanent filter should also get used by all others query, except insert and batchInsert, where it has no sense.